frontend/package.json

```diff
@@ -119,5 +119,9 @@
   },
   "browserslist": [
     ">0.25% and last 2 years"
-  ]
+  ],
+  "devDependencies": {
+    "@mdi/font": "^7.4.47",
```

Comment:
Due to some breaking changes in the [material icons font](https://www.npmjs.com/package/material-design-icons-iconfont) that we use, we added the required CSS and font files to our own repository under [`/frontend/src/css/vendor/icons`](https://github.com/photoprism/photoprism/tree/develop/frontend/src/css/vendor/icons) (AFAIK, the original package no longer included a plain CSS file or the font paths in it were invalid, so the icons broke in the UI after updating the package):
- https://www.npmjs.com/package/material-design-icons-iconfont
- https://jossef.github.io/material-design-icons-iconfont/

Instead of using/importing the original npm package, `/frontend/src/css/vendor/iconsmaterial-design-icons.css` is imported in our main CSS file:
https://github.com/photoprism/photoprism/blob/0b391d26ecf3989becef2b8280a15b0014e709f2/frontend/src/css/app.css#L26

If Vuetify 2/3 use `@mdi/font` instead of `material-design-icons-iconfont` then that's fine with us as both should support/include similar icons since they are also based on Material Design:
- https://fonts.google.com/icons

Note, however, that the naming of some icons MAY be different and that you will therefore probably need to change some templates so they use the new Material Design font icons:
- https://www.npmjs.com/package/@mdi/font
- https://pictogrammers.com/library/mdi/

Also, would it make sense to list `@mdi/font` under the regular `dependencies` in the `package.json` file, since the fonts are always required? See the comment below.

Reply:
Yes, you are right.
I will move `@mdi/font` package to the `dependencies` because we always need them.